### PR TITLE
update AngleSharp and AngleSharp.Css

### PR DESCRIPTION
**Pull request by SimonCropp**
so the nugets r all in sync

### CODE REVIEW

**Comment by egil** [comment]
Yeah I've been waiting for the css package to move to V1. Looks like we're close.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 0.19.0
```
Comment (egil) [inline CHANGELOG.md:1]:
```suggestion
# 1.0.0
```

**Review by egil — CHANGES_REQUESTED** [review]
No reason not to go to V1 here too

**Comment by egil** [comment]
Hmm now that I think about it, I do not think we can push a none-preview version out which depends on preview packages? so perhaps lets version it as 1.0.0-preview until CSS package is v1. @FlorianRappl what do you think?

**Comment by FlorianRappl** [comment]
Well we could do it (publish something that is based on a preview) but I fully agree that waiting for AngleSharp.Css 1.0.0 is the right move!

**Comment by egil** [comment]
I keep running into issues with wanting to use AngleSharp v1 and not being able to, so pushed an updated that ignores the preview release of css. Diffing doesn't need all css features, in realilty just the parsing of `style` attributes content, so I feel ok with this.

**Pull request closed without merging**